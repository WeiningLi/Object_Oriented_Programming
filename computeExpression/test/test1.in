3 x + 5 - y * 6 / ABS NEG done
eval
set y 2
eval 
copy
set x 3
eval
print 
copy
unset y
copy
set y 2
copy
unset x
copy

