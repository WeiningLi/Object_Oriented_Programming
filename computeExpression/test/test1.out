-|((((3 + x) - 5) * y) / 6)|
x has no value.
x has no value.
-|((((3 + x) - 5) * 2) / 6)|
-|((((3 + 1) - 5) * 2) / 6)|
0
0
-|((((3 + 3) - 5) * 2) / 6)|
-|((((3 + 3) - 5) * 2) / 6)|
-|((((3 + 1) - 5) * 2) / 6)|
0
-|((((3 + 3) - 5) * y) / 6)|
-|((((3 + 1) - 5) * y) / 6)|
y has no value.
-|((((3 + 3) - 5) * 2) / 6)|
-|((((3 + 1) - 5) * 2) / 6)|
0
-|((((3 + x) - 5) * 2) / 6)|
-|((((3 + 1) - 5) * 2) / 6)|
0
